awx/main/management/commands/register_peers.py

```diff
@@ -0,0 +1,31 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from awx.main.models import Instance, InstanceLink
+
+
+class Command(BaseCommand):
+    """
+    Internal tower command.
+    Register the peers of a receptor node.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('source', type=str, help="")
+        parser.add_argument('--peers', type=str, nargs='+', required=True, help="")
```

Comment:
Recording here what was said offline: no, it does not make the list absolutely true (i.e. all links _not_ in the `peers` parameter would be deleted).  I would oppose such behavior, since it would make it impossible to incrementally add links.

I absolutely should add help text, though.  :facepalm: